assets/css/style.css

```diff
@@ -1,4 +1,6 @@
 @import "https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css";
+@import url('https://fonts.googleapis.com/css?family=Inconsolata:700');
+@import url("https://fonts.googleapis.com/css?family=Inconsolata:700");
 
```

Comment:
I have reviewed the merge request with Tyler and we are suggesting the following changes.  (overall, the code looks okay, but it is disorderly an example, the footer stylings should be at the bottom).

More individual comments lower.